karaage/people/forms.py

```diff
@@ -92,6 +92,17 @@ class Meta:
             'address', 'country'
         ]
 
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        users = Person.objects.filter(email__exact=email)
+        if users.count() > 0:
+            raise forms.ValidationError(
+                six.u(
+                    'An account with this email already exists. '
+                    'Please email %s')
+                % settings.ACCOUNTS_EMAIL)
```

Comment:
An easier way might to be replace:

```
Person.objects.filter(email__exact=email)
```

with:

```
Person.objects.filter(email__exact=email).exclude(pk=...)
```

I think the current pk of the user should be available somewhere, I can't remember where offhand. This pattern elsewhere in Karaage. Have a look at ``karaage/people/utils.py`` for some examples.